### PR TITLE
Install vagrant-triggers plugin if it not installed

### PR DESCRIPTION
**Pull request by bborysenko**
Just a minor addition for #102 to automatically install vagrant-trigger plugin, it it has not been yet installed

### CODE REVIEW

File: molecule/templates/vagrantfile.j2 [hunk 1]
```diff
@@ -1,3 +1,13 @@
+{%- for platform in config.vagrant.platforms %}
+  {%- if platform.name == current_platform %}
+    {%- if 'triggers' in platform %}
+unless Vagrant.has_plugin?('vagrant-triggers')
+    system "vagrant plugin install vagrant-triggers"
```
Comment (abrown-sg) [inline molecule/templates/vagrantfile.j2]: Should be two spaces instead of 4.  Looks good otherwise!

**Comment by bborysenko** [comment]
Text indentation fixed

**Comment by abrown-sg** [comment]
:+1:

**Comment by retr0h** [comment]
:+1:

**Pull request merged**